apps/nextra/pages/en/build/cli/running-a-local-network.mdx

```diff
@@ -96,7 +96,7 @@ called \`Result::unwrap()\` on an \`Err\` value: Os { code: 24, kind: Uncategori
 This means there were too many open files on your system. On many Unix systems you can increase the maximum number of open files by adding something like this to your `.zshrc`:
 
 ```bash filename="Terminal"
-ulimit -n 32768
```

Comment:
the local network doesn't need so many fds, but I guess it's okay to just make them consistent.

we actually use the number 1048576 in many places in the codebase as I searched, maybe we should use that number?